competitions/s1floods/benchmark_tutorial.ipynb

```diff
@@ -0,0 +1,2029 @@
+       "3  awc01_vv   awc01      awc           vv  Bolivia  2018-02-15\n",
+       "4  awc02_vh   awc02      awc           vh  Bolivia  2018-02-15"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_metadata.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The training data for this competition consist of two [polarization bands](https://nisar.jpl.nasa.gov/mission/get-to-know-sar/polarimetry/) that bring out distinct physical properties in a scene: VV (vertical transmit, vertical receive) and VH (vertical transmit, horizontal receive). Each band is identified by a unique `image_id`, which is equivalent to `{chip_id}_{polarization}`. A `chip_id` consists of a three letter `flood_id` and a two digit chip number. **Each unique chip has both a `_vv` and `_vh` band.** For example, `awc00_vh` represents the `vv` band of chip number `00` for flood `awc`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(1084, 6)"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_metadata.shape"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "542"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_metadata.chip_id.nunique()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's take a look at how many flood events are in the training data and how many chips we have per event."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "count    13.000000\n",
+       "mean     41.692308\n",
+       "std      19.947367\n",
+       "min      15.000000\n",
+       "25%      28.000000\n",
+       "50%      32.000000\n",
+       "75%      65.000000\n",
+       "max      69.000000\n",
+       "Name: chip_id, dtype: float64"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "flood_counts = train_metadata.groupby(\"flood_id\")[\"chip_id\"].nunique()\n",
+    "flood_counts.describe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The training data include chips from 13 flood events. We have anywhere from 15 to 69 chips (30 to 138 images) per unique event, with half of events containing fewer than 32 chips (64 images).\n",
+    "\n",
+    "We can also take a look at the distribution of chips by location."
```

Comment:
We should have seaborn installed, in which case the next cell is something like `sns.countplot(x="location", y="chip_id")` (untested).